vagrant_setup/go.sh

```
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+sudo curl -O https://storage.googleapis.com/golang/go1.18.1.linux-amd64.tar.gz
+sudo tar -xvf go1.18.1.linux-amd64.tar.gz
+sudo mv go /usr/local
+sudo echo 'export GOROOT=/usr/local/go' >> /etc/profile
+sudo echo 'export PATH=$PATH:/usr/local/go/bin' >> /etc/profile
+sudo echo 'export GOPATH=/usr/local' >> /etc/profile
+sudo echo 'export PATH=$PATH:$GOPATH/bin' >> /etc/profile
+sudo echo 'export GO111MODULE=on' >> /etc/profile
+sudo echo 'export CGO_ENABLE=0' >> /etc/profile
```

Shellcheck (0.10.0)

[warning] sudo doesn't affect redirects. Use .. | sudo tee -a file

(SC2024)